chrome/content/zotero/xpcom/utilities_internal.js

```diff
@@ -1700,6 +1700,52 @@ Zotero.Utilities.Internal = {
 		elem.appendChild(menu);
 		return menu;
 	},
+
+	// Show View > Columns, Sort By and Move Column menus for windows that have an itemTree
+	handleItemTreeMenuShowing: function (event, menuID, itemsView) {
```

Comment:
That makes sense! The only reason why I initially put it into the utilities is because I couldn't think of a better place to share functionality. But static methods on `ItemTreeMenuBar` is a way more logical spot, especially now that there are only 2 places where these methods are used (`zoteroPane` and `ItemTreeMenuBar.init()`).